docs/content/dagster-plus/managing-deployments/alerts/slack.mdx

```diff
@@ -32,9 +32,13 @@ To follow the steps in this guide, you'll need:
 
 ---
 
-## Step 2: Invite the Dagster+ bot to Slack channels
+## Step 2: Add the Dagster Cloud bot to Slack channels
```

Comment:
Do you know if it's possible to rename the bot instead? I feel like this is a relic of the old naming.

Reply:
I don't, I'm wary it could be a breaking change minefield